### ISSUE TITLE
*.cur and/or *.ani support

### ISSUE THREAD

**Issue by wlwl2** [aseprite/aseprite#1976]
Add support for exporting to .cur and/or .ani filetypes. I suppose this enhancement would be low priority, nevertheless it is a good suggestion.

Discussion at:
https://community.aseprite.org/t/cur-and-or-ani/2391

**Comment by FFiesta93** [comment]
I'm looking forward to this. In this very particular case though, it needs a way to point out where would the "clicking point" be (e.g. text cursors, crosshairs, hand, etc.).

**Comment by ElderlyGlek** [comment]
Yes, support for cursor formats is definitely as necessary a feature as creating your own fonts directly in Aseprite (somewhere I met a feature request about this). Aseprite is by far the best program for pixelated artists, but it will get even better (especially for game developers) with the above features.

**Comment by dacap** [comment]
Personally I'd love to have full .ico/.cur/.ani support on Aseprite, as the last time I tried to modify the Aseprite icon on Windows was a nightmare.

I'll give some priority to this feature for the following months.
